app/globals.css

```
@@ -96,6 +96,12 @@
 		outline-color: color-mix(in oklab, var(--ring) 50%, transparent);
 	}
 
+	::selection,
+	::-moz-selection {
```

Stylelint (17.11.0)

[error] Vendor-prefixed selector "::-moz-selection" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)